README.md

```diff
@@ -84,6 +84,10 @@ you can see this workflow in action on our [demo repo](https://github.com/Bearer
 
 **Optional** Specify which format to use, e.g. `json`
 
+### `output`
+
+**Optional** Specify where to store the report, e.g. `results.sarif`
```

Comment:
do wonder if we should get rid of `outputs.rule_breaches` and just use file output instead much easier to handle and we can get rid of the escape values. WDYT?

Reply:
Let's keep it for now. I agree that maybe it's overkill but it doesn't harm to have it for now

Reply:
ok would require a v3 anyway probably 🤔